task/hadolint/0.1/hadolint.yaml

```diff
@@ -0,0 +1,59 @@
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  annotations:
+    tekton.dev/categories: Code Quality
+    tekton.dev/displayName: Hadolint
+    tekton.dev/pipelines.minVersion: 0.12.1
+    tekton.dev/platforms: linux/amd64
+    tekton.dev/tags: 'Kubernetes, Misconfiguration'
+  name: hadolint
+  labels:
+    app.kubernetes.io/version: '0.1'
+spec:
+  description: >-
+    This task makes it possible to use Hadolint within Tekton Pipeline.
+
+    A smarter Dockerfile linter that helps you build best practice Docker
+    images. The linter parses the Dockerfile into an AST and performs rules on
+    top of the AST
+  params:
+    - default: ''
+      description: ignore rules.
+      name: ignore-rules
+      type: string
+    - default: './Dockerfile'
+      description: Dockerfile path.
+      name: dockerfile-path
+      type: string
+    - default: tty
+      description: >-
+        The output format for the results [tty | json | checkstyle | codeclimate
+        | gitlab_codeclimate | codacy] (default tty).
+      name: output-format
+      type: string
+  steps:
+    - image: 'ghcr.io/hadolint/hadolint:v2.8.0-alpine'
```

Comment:
can we add a digest to the image as well. example `docker.io/alpine/git:v2.26.2@sha256:23618034b0be9205d9cc0846eb711b12ba4c9b468efdd8a59aac1d7b1a23363f`

Reply:
Fixed.
` ~  podman inspect --format='{{index .RepoDigests 0}}' ghcr.io/hadolint/hadolint:v2.8.0-alpine                                                                                                                                                              ghcr.io/hadolint/hadolint@sha256:6c4b7c23f96339489dd35f21a711996d7ce63047467a9a562287748a03ad5242`